README.md

```diff
@@ -1,76 +1,72 @@
-
 # 10 Tips to Get or Keep Your GitHub Ready for Job Seeking (and Bonus Tips to Make it Unstoppable)
 
-Github isn't just a repository for code — it’s another part of your online presence, and arguably the most specific tool a hiring manager has (without talking to you) to understand how you’ll stack up as a candidate. Based on _**numerous**_ feedback sessions with employers we **know** that your GitHub profile will be a critical step in their assessment of you as a candidate. 
-
-**Note:** Sometimes when graduates leave the program, they stop receiving attribution for their Git commits (i.e. the link between their computer and their GitHub account becomes severed). This can happen for a variety of reasons. To ensure the commits you make on your computer are associated to your GitHub profile, follow the steps below:
+Github isn't just a repository for code — it’s another part of your online presence, and arguably the most specific tool a hiring manager has (without talking to you) to understand how you’ll stack up as a candidate. Based on _**numerous**_ feedback sessions with employers we **know** that your GitHub profile will be a critical step in their assessment of you as a candidate.
 
+> **Note:** Sometimes when graduates leave the program, they stop receiving attribution >for their Git commits (i.e. the link between their computer and their GitHub account >becomes severed). This can happen for a variety of reasons. To ensure the commits you >make on your computer are associated to your GitHub profile, follow the steps below:
```

Comment:
```suggestion
> **Note:** Sometimes when graduates leave the program, they stop receiving attribution >for their Git commits (i.e. the link between their computer and their GitHub account becomes severed). This can happen for a variety of reasons. To ensure the commits you make on your computer are associated to your GitHub profile, follow the steps below:
```